### ISSUE TITLE
Login and signup page for the design 

### ISSUE THREAD

**Issue by huamanraj** [huamanraj/Covid_website#2]
create a login and signup page for the website using HTML and CSS

**Comment by Surajit0573** [comment]
Hi @huamanraj, I can create a beautiful login and signup page for this website. Can you please assign me to this issue !!

**Comment by huamanraj** [comment]
yep! do it but take care of design  that align with website and responsiveness!!

**Comment by huamanraj** [comment]
> Hi! I would like to help this project. But is there the figma template from [Design.png] ?

sorry but there is no figma file of the design!

**Comment by DylanP97** [comment]
too bad! 
let me know if you need help anyway with the website design
I'm available

**Comment by huamanraj** [comment]
> too bad! 
> let me know if you need help anyway with the website design
> I'm available

Sure!
